File: 3811-reverse-degree-of-a-string/3811-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        l = list('abcdefghijklmnopqrstuvwxyz')
        Sum = 0
        for index,i in enumerate(s):
            Sum+=((26-l.index(i))*(index+1))
            #print(26-l.index(i))
        return Sum

File: 3811-reverse-degree-of-a-string/3811-reverse-degree-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var reverseDegree = function(s) {
    let Summ = 0;
    let l = 'abcdefghijklmnopqrstuvwxyz'.split('');

    for (let index = 0; index < s.length; index++) {
        let i = s[index];
        let j = l.indexOf(i);

        Summ+=(26-j)*(index+1)
    }
    return Summ
}